### DISCUSSION TITLE
Se você pudesse voltar 10 anos, o que diria pra si mesmo?

### DISCUSSION THREAD

**Discussion by TailoMateus** [frontendbr/forum#955] [Discussão]
Pessoal, fiz essa mesma pergunta no dev.to, e gerou algumas boas respostas. 

Segue o link: https://dev.to/tailomateus/if-you-could-go-back-10-years-what-would-you-say-to-yourself--1k6l

Acredito que também podemos discutir aqui.

**Comment by renanlara** [comment]
Aprenda JavaScript.

**Comment by kelciocajueiro** [comment]
Compre bitcoin e seja feliz.

**Comment by kivervinicius** [comment]
Certeza que seria, invista em bitcoin (apesar que do jeito que sou azarado, certeza que não teria valorizado nada, kkk)

**Comment by kazzkiq** [comment]
Esqueça esse negócio de web. Invista em botijão de gás, gasolina, e bitcoin. 😄 


----
**Edit:**
Agora falando sério, eu diria pra começar aprendendo JavaScript de verdade. A fundo. Sem jQuery. Isso ia me poupar muita dor de cabeça e oportunidades perdidas no futuro...

**Comment by mariojvr** [comment]
Tenho 27 anos e diria: Guarde dinheiro! Guarde dinheiro! Guarde dinheiro!

**Comment by renanlara** [comment]
@mariojvr, investir seria melhor que guardar.

**Comment by mariojvr** [comment]
@renanlara Concordo. Mas se eu conseguisse o mínimo que é guardar, eu já teria ótimos resultados.

**Comment by luizbills** [comment]
Eu diria: "Não perde seu tempo estudando java".

**Comment by jandrei** [comment]
Diria, não passe em concurso, seja empreendedor.

**Comment by felipefialho** [comment]
"Aprende inglês a todo custo"

**Comment by yurischulz** [comment]
Para de matar aula e desperdiçar teu tempo com nadismos, se esforça e estuda! Não tem erro.
Ah, não pegue freelas a não ser que precise MUITO, ao invés disso: estude!

**Comment by fabiomirandaa** [comment]
Aprenda inglês, esquece essa merda que te falaram de que programação num dá futuro e que concurso que dá, e invista em Bitcoin e venda quando ele estiver valendo 69K.

**Comment by pedrocss** [comment]
- Esqueça a linguagem do momento, estude os conceitos por trás das tecnologias;
- Aprenda inglês o mais rápido possível.

**Comment by bernardodiasc** [comment]
> Vai ficar tudo bem!

**Comment by shadowmaru** [comment]
Não precisa se matar de trabalhar pra "mostrar serviço". Se você for bom, vai ter quem te valorize sem te explorar (difícil, mas tem).

Mais uma: comunidades são a melhor porta de entrada para os melhores empregos (pelo menos aqui no Brasil).

**Comment by Murillofilho86** [comment]
primeiro: Ouça o seu pai! 
Segundo: Compre bitcoins

**Comment by ghost** [comment]
invista em aprender inglês
pare de ficar na internet fazendo vários nadas 
pare de ouvir seus amiguinhos que não sabem o que fazer da vida porque aos 25 anos eles vão estar num limbo pior que o teu
estude pacas hoje e não terás que estudar tanto no futuro 
pare de discutir na internet pq é isso que está te dando crise de ansiedade

**Comment by karluslima** [comment]
Aprende Inglês, Javascript desde o começo, compre bitcoins, vá pra academia "vagal" : P

**Comment by florbraz** [comment]
Pro lado profissional, vou na linha do "aprenda javascript, não só jquery".

Mas pra vida em geral, principalmente: "Guarde dinheiro." "Estrague menos seu corpo ingerindo/usando porcaria." "Valorize quem te valoriza, seja no campo profissional, da amizade, do amor ou da família."

**Comment by kvnol** [comment]
Não consigo participar da brincadeira, 10 anos atrás eu nem tinha computador ainda, hahahaha. Se fosse 5 anos atrás, seria "Aprenda JavaScript e não foque só no design das coisas", com toda certeza.

**Comment by yanmagale** [comment]
- Valorize as pessoas que estão perto de você
- Fortaleça a qualidade dos seus pensamentos
- Se preocupe apenas com a sua prática e melhoria diárias

**Comment by shadowmaru** [comment]
> eu diria: VÁ PEGAR ALGUEM E PARE COM ESSE NEGOCIO DE VIDEOGAMES, leite com pera
> Talvez faria algum bullying mas de leve tbm

Não sei o que isso tem a ver com carreira.

**Comment by caioincau** [comment]
Pessoal: Compre Bitcoin, invista mais na saúde emocional e fisíca.
Profissional: Estude outra área e faça pós/cursos soltos voltados para programação, talvez economia.

**Comment by mateusmiguel** [comment]
Pare de procrastinar e estude lógica de uma vez. 
Preste alguma atenção nas aulas da escola. Quando você crescer vai sentir falta de ter um professor do seu lado todo dia.
Ouça seus pais.

Você é canhoto mas o mundo não. Aprenda a tocar guitarra e baixo do lado certo.
Deixe o cabelo crescer.
Pare de colocar ketchup em tudo. 

~~**Em tempo:** invista em gasolina.~~
